assets/js/storage.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
 module.exports = {
-    get(name, fallback) {
-        const result = localStorage.getItem(name);
+    get(name, fallback, storage = localStorage) {
```

Comment:
it feels weird that the fn takes the whole storage object in as a parameter. i believe it would be better if the parameter was a string - "local" or "session" - and these functions could choose the correct storage based on that string. then the storages would only be accessed in these methods and it would be less error-prone since we wouldn't allow completely custom storage objects to be passed in.

Reply:
✅